AssemblyLoadDebuggerShared/ToolWindow/AssemblyLoadDebuggerToolWindow.cs

```diff
@@ -1,15 +1,15 @@
-﻿namespace AssemblyLoadDebugger
-{
-    using System;
-    using System.Runtime.InteropServices;
-    using Microsoft.VisualStudio.Shell;
+﻿using System.Runtime.InteropServices;
+using AssemblyLoadDebugger;
+using Microsoft.VisualStudio.Shell;
 
+namespace ToolWindow
+{
     [Guid("71604717-b15d-4d1e-9fb4-03f5122e8858")]
     public class AssemblyLoadDebuggerToolWindow : ToolWindowPane
     {
         public AssemblyLoadDebuggerToolWindow() : base(null)
         {
-            this.Caption = Vsix.Name;
+            this.Caption = "Assembly Load Debugger";
```

Comment:
@mlorbetske I can see that you were generating the Vsix class based on the manifest file.
In order to target VS2022 you need a different manifest file per target so I couldn't use that in the shared code.
... I couldn't figure out how to generate the static class either, but the above point means that is moot